lib/karafka/railtie.rb

```diff
@@ -136,6 +114,18 @@ class Railtie < Rails::Railtie
           end
         end
       end
+
+      initializer 'karafka.configure_worker_external_executor' do |app|
+        app.config.after_initialize do
```

Comment:
I don't love putting this in an `after_initialize` callback, but I needed the `consumer_persistence` value after loading the app's boot file.